mne/epochs.py

```diff
@@ -1864,6 +1864,35 @@ def equalize_event_counts(self, event_ids, method='mintime'):
         # actually remove the indices
         return self, indices
 
+    def shift_time(self, tshift, relative=True):
+        """Shift time scale in epoched data.
+
+        Parameters
+        ----------
+        tshift : float
+            The amount of time shift to be applied if relative is True
+            else the first time point. When relative is True, positive value
+            of tshift moves the data forward while negative tshift moves it
+            backward.
+        relative : bool
+            If true, move the time backwards or forwards by specified amount.
+            Else, set the starting time point to the value of tshift.
+
+        Notes
+        -----
+        Maximum accuracy of time shift is 1 / epochs.info['sfreq']
```

Comment:
The name of the method and the docstring are the same as in `Evoked`. I think it is important to maintain consistency there.

I agree that both the method name and the docstring could be clearer. But I think this would be quite a drastic API change especially as it would not be backward compatible or it would break the logical consistency between `Evoked` and `Epochs` which I think is an important issue for the users.

Reply:
Ahh bummer, I didn't realize we had it there. Yes maintain consistency and we can fix both later at some point